src/components/Navigation.spec.js

```diff
@@ -0,0 +1,121 @@
+import { render, screen } from '@testing-library/react'
+import { MemoryRouter as Router } from 'react-router-dom'
+import userEvent from '@testing-library/user-event'
+import Navigation from './Navigation'
+import placeholder from '../images/placeholder.png'
+
+describe('Navigation', () => {
+  const books = [
+    {
+      id: '1',
+      volumeInfo: {
+        title: 'Harry Potter and the Goblet of Fire',
+        authors: ['J. K. Rowling'],
+        imageLinks: {
+          thumbnail: placeholder,
+        },
+      },
+      finished: false,
+      readingSince: '05/2021',
+      finishedSince: '',
+    },
+    {
+      id: '2',
+      volumeInfo: {
+        title: 'Cathedral of the Sea',
+        authors: ['Ildefonso Falcones de Sierra'],
+        imageLinks: {
+          thumbnail: placeholder,
+        },
+      },
+      finished: true,
+      readingSince: '05/2021',
+      finishedSince: '08/2021',
+    },
+  ]
+
+  it('renders two links', () => {
+    render(
+      <Router>
+        <Navigation />
+      </Router>
+    )
+
+    const links = screen.getAllByRole('link')
+    expect(links).toHaveLength(2)
+  })
+
+  it('changes to active when "currently reading" link is clicked', () => {
+    const mockOnHandleActiveReadingStatus = jest.fn()
+
+    render(
+      <Router>
+        <Navigation
+          books={books}
+          onHandleActiveReadingStatus={mockOnHandleActiveReadingStatus}
```

Comment:
I would add to the test, that when you click on it, that `mockOnHandleActiveReadingStatus` (which is quite a long name btw) is called with the reading status and books.